src/validations/rules/ssp.sch

```diff
@@ -2874,7 +2874,7 @@ leveraged-authorization.</sch:assert>
                         id="user-has-authorized-privilege-diagnostic">This user lacks one or more authorized-privileges.</sch:diagnostic>
         <sch:diagnostic doc:assertion="user-user-type-has-allowed-value"
                         doc:context="oscal:user/oscal:prop[@name = 'type']"
-                        id="user-user-type-has-allowed-value-diagnostic">User type property has an allowed value.</sch:diagnostic>
+                        id="user-user-type-has-allowed-value-diagnostic">This user type property lacks an allowed value.</sch:diagnostic>
```

Comment:
The framing is different here than the other diagnostic changes - should it be along the lines of "user type properties must only include allowed values"?

Reply:
Multiple diagnostic messages (the ones accompanying failed assertions) were not appropriate (they essentially duplicated the affirmative message). The assertion message should be an affirmative one (successful test being an affirmation of the messages) and a failed assertion message (diagnostic) should be a description of the deficiency in the context of the rule.

Reply:
This was somewhat related to https://github.com/18F/fedramp-automation/issues/97.